.travis.yml

```diff
@@ -1,13 +1,41 @@
+dist: bionic
 language: java
-
+services:
+  - docker
 jdk:
   - openjdk11
   - oraclejdk11
-
 before_cache:
   - rm -f  $HOME/.gradle/caches/modules-2/modules-2.lock
   - rm -fr $HOME/.gradle/caches/*/plugin-resolution/
 cache:
   directories:
-    - $HOME/.gradle/caches/
-    - $HOME/.gradle/wrapper/
\ No newline at end of file
+    - "$HOME/.gradle/caches/"
+    - "$HOME/.gradle/wrapper/"
+before_install:
+  - docker pull m0rf30/pacur-archlinux
+  - docker pull m0rf30/pacur-fedora-31
+  - docker pull m0rf30/pacur-opensuse
+  - docker pull m0rf30/pacur-ubuntu-eoan
+script:
+  - cd pacur
+  - docker run -v $(pwd):/pacur -ti m0rf30/pacur-archlinux
+  - docker run -v $(pwd):/pacur -ti m0rf30/pacur-fedora-31
+  - docker run -v $(pwd):/pacur -ti m0rf30/pacur-opensuse
+  - docker run -v $(pwd):/pacur -ti m0rf30/pacur-ubuntu-eoan
+  - for i in $(ls); do sha256sum $i >> sha256sums; done
+deploy:
+  provider: releases
+  api_key:
+    secure: ceDHwwztvMJexkdnLG4h7PFGEZ3K+rpdkzOZ+qnzMzpo+ljeWGEQhHOS9QurLEraa7ncv2c7nMbdTKMorn1afKkOi0l3aT6rHZfZyGbffaXg59M/4wIdm90tdYZYvSE5pg5A0KGk15/oKenqpbSWaQTqbh2xIxDde5lxGs8lmQlOKE9dKaf16ue/swP84dT9xHFDMLR+t6GcRa4a7SZO/jgO4LrdiDhn25kI9lU7KFv1B5sB4+5CRwfq36Gj8J8eu/gHvL5T0z6vrG2HQT0QlvPUiPBS7OIa7lR2VMsVT8I/PO6kjLxduEUcuINVjkEJ5NqsDkMEPSLv+QOcgucz2QmV3rsZPl4+70GyvIc8xzqaaradcubbBxUNJpamCqg1PnFBfpvs5Aok+XsQ42m0Jw0Fv8sxjcVTPFRZGzufJ+txkeOZKK5nncfKEhA7MMpACs3kXQ/xHZIjdoGSSKBiglYMG25FYaB09qCT877EpeXhKziJc5HMWHv3RyYgy8dJDra0gX9hm9LrCU8Hc5nWOks8jIhFgyhHtzXe1248Xh3rQVgd2TtcsofVyLLTyUbBrqAliybiZtl2+licYdA+t5pAkH27E+yXU9Z0D8xuzNjY5UXGRA2tsK9nUzWJkVd5NmE5ceahncUShMRuLn1Q+VhlWW8LqPqCQPcNtoo3lB0=
+  file_glob: true
+  file: ./*.deb
+  file: ./*.rpm
+  file: ./*.tar.zst
+  file: ./sha256sums
+  skip_cleanup: true
+  prerelease: true
+  on:
+    repo: Polpetta/jkk
+    branch: master
+    tags: false
```

Comment:
I'd push binaries only when we create tags, so that a tag corresponds to an official release!

That said, I'd change `prelease` from `true` for `false`. If the user really wants to try the latest version he can install AUR and use the `jkk-git` package :wink: :joy:

Reply:
Done. Waiting for a sparkling release tag :dancer: